### DISCUSSION TITLE
doc: do we need to sort deployment by language

### DISCUSSION THREAD

**Discussion by elsatch** [open-webui/docs#141] [General]
I just wonder if it would be benefitial to add language sub headers (English, Spanish, Hindi, etc) in the future, in case it keeps growing.

**Comment by tjbck** [comment]
docusaurus has built-in [i18n support](https://docusaurus.io/docs/i18n/introduction)!

**Comment by ArthurDarkstone** [comment]
language sub headers added on [https://github.com/open-webui/docs/pull/54](https://github.com/open-webui/docs/pull/54)

**Comment by ArthurDarkstone** [comment]
the problem is how to keep all kinds of language docs updated. For example, the current PR is an update to English doc and cannot be updated to other languages.

**Reply by ArthurDarkstone** [reply]
use Crowdin maybe a good idea

[https://docusaurus.io/docs/i18n/crowdin](https://docusaurus.io/docs/i18n/crowdin)
